src/plugins/boost/screens/sponsor/Form.jsx

```diff
@@ -7,15 +7,22 @@ import History from '@triniti/cms/plugins/pbjx/components/history';
 import RawContent from '@triniti/cms/components/raw-content';
 import SponsorFields from '@triniti/cms/plugins/boost/components/sponsor-fields';
 import Message from '@gdbots/pbj/Message';
-import StreamId from '@gdbots/schemas/gdbots/pbjx/StreamId';
+import setBlocks from '@triniti/cms/plugins/blocksmith/utils/setBlocks';
 
 import schemas from './schemas';
 
-const Form = ({ node: sponsor, tab, isEditMode }) => {
-  const streamId = StreamId.fromString(`sponsor.history:${sponsor.get('_id')}`);
+const Form = ({ form, node: sponsor, tab, isEditMode }) => {
   switch (tab) {
     case 'history':
-      return <History schema={schemas.getNodeHistoryRequest} streamId={streamId} />;
+      return (
+        <History
+          isEditMode={isEditMode}
+          formName={form}
+          node={sponsor}
+          schema={schemas.getNodeHistoryRequest}
+          setBlocks={setBlocks}
```

Comment:
@rsumilang same as poll. i think only people, pages, and articles can have blocks